### PR TITLE
Changed db to postgis

### PR DESCRIPTION
**Pull request by Meisterschueler**

### CODE REVIEW

**Comment by Meisterschueler** [comment]
Todo: pypi 0.2.1 --> 0.3.0 ?

File: alembic/versions/277aca1b810_migrate_to_postgis.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+DOWNGRADE_QUERY = """
+UPDATE {table_name}
+SET
+    latitude = ST_Y(ST_TRANSFORM(location, 4326)),
+    longitude = ST_X(ST_TRANSFORM(location, 4326));
+"""
+
+def upgrade():
+    #CREATE EXTENSION IF NOT EXISTS postgis
```
Comment (kerel-fs) [inline alembic/versions/277aca1b810_migrate_to_postgis.py]: The migration script could run this command by itself (do I overlook something?):
```python
op.execute("CREATE EXTENSION IF NOT EXISTS postgis;");
```

File: alembic/versions/277aca1b810_migrate_to_postgis.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    op.drop_column('receiver', 'longitude')
+
+    op.add_column('takeoff_landing', sa.Column('location', ga.Geometry('POINT', srid=4326)))
+    op.execute(UPGRADE_QUERY.format(table_name='takeoff_landing'))
+    op.drop_column('takeoff_landing', 'latitude')
+    op.drop_column('takeoff_landing', 'longitude')
+
+
+def downgrade():
+    #DROP EXTENSION postgis
```
Comment (kerel-fs) [inline alembic/versions/277aca1b810_migrate_to_postgis.py]: as above,
```python
op.execute("DROP EXTENSION postgis;");
```

File: tests/test_utils.py [hunk 3]
```diff
@@ -45,6 +45,10 @@ def test_get_country_code_bad(self):
         country_code = get_country_code(latitude, longitude)
         self.assertEqual(country_code, None)
 
+    def test_get_airports(self):
```
Comment (kerel-fs) [inline tests/test_utils.py]: > Intuitively, one can view a unit as the smallest testable part of an application. - [Unit testing, WP](https://en.wikipedia.org/wiki/Unit_testing)

This test is neither minimal, nor does it test our code (it mostly tests the seeyou-parser of aerofiles instead).
Imho we should drop it entirely, or at least reduce the test input to a minimal set of entries (1?).

**Pull request merged**